api-extractor/src/generators/ApiFileGenerator.ts

```diff
@@ -23,6 +23,12 @@ import { ApiTag } from '../definitions/ApiDocumentation';
   */
 export default class ApiFileGenerator extends ApiItemVisitor {
   protected _indentedWriter: IndentedWriter = new IndentedWriter();
+  /**
+   * We don't want to require documentation for any properties that occur
+   * anywhere within a TypeLiteral. If this value is above 0, then we are 
+   * visiting something within a TypeLiteral. 
+   */
+  protected _insideTypeLiteral: number;
```

Comment:
Protected members do not start with _.

This seems like it's actually private.